lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -214,7 +214,7 @@ private static function get_inner_blocks_from_fallback( $attributes ) {
 
 		// Fallback my have been filtered so do basic test for validity.
 		if ( empty( $fallback_blocks ) || ! is_array( $fallback_blocks ) ) {
-			return '';
+			return new WP_Block_List( array(), $attributes );
```

Comment:
Confirmed present in new block-library version of this file

https://github.com/WordPress/gutenberg/blob/e0005a813686157e2f25f6d4b28cc72ebadbfaae/packages/block-library/src/navigation/index.php#L218-L221